mkdocs.yml

```diff
@@ -155,7 +155,9 @@ nav:
     - './examples/index.md'
     - './examples/Ch-ExamplesVirtualDeviceService.md'
     - './examples/Ch-ExamplesSendingAndConsumingBinary.md'
-    - './examples/AppServiceExamples.md'
+    - App Service Examples:
+      - './examples/AppServiceExamples.md'
```

Comment:
The camera management app in this file still points to the doc in the example app repo, are we not planning to remove it?

Reply:
The table in the AppServiceExamples file is supposed to point to the repo itself